gnc/sub8_perception/nodes/bins_2d.py

```diff
@@ -53,8 +54,10 @@ def __init__(self):
         self.last_image_time = None
         self.camera_model = None
         self.pose_service = rospy.Service('vision/bin/2D', VisionRequest2D, self.request_bin)
-        self.image_sub = sub8_ros_tools.Image_Subscriber('/down_camera/image_rect_color', self.image_cb)
+        self.image_sub = sub8_ros_tools.Image_Subscriber('/down/left/image_raw', self.image_cb)
```

Comment:
If you wanna test with a bag, you should run[ image_proc/rectify](http://wiki.ros.org/image_proc).